CorinneTest/IPerson.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace CorinneTest
+{
+    public interface IPerson
+    {
+        string Name();
+        int Age();
+        //Person.Gender PersonGender();
```

Comment:
probably need to replace this with the new `Gender` that isn't nested in `Person`

Reply:
yes that makes sense. I'm deleting IPerson and IPersonFactory interfaces because they aren't necessary for this project.